examples/cloudflare-workers/package.json

```diff
@@ -14,5 +16,7 @@
   "dependencies": {
     "@upstash/ratelimit": "^1.0.3",
     "@upstash/redis": "^1.20.6"
-  }
-}
+  },
+  "module": "index.ts",
+  "type": "module"
```

Comment:
why do we need these?

Reply:
`bun init` command probably adds them automatically. removed